src/GraphQL/SortBlockMutationCreator.php

```diff
@@ -0,0 +1,93 @@
+<?php
+namespace DNADesign\Elemental\GraphQL;
+
+use DNADesign\Elemental\Models\BaseElement;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Definition\Type;
+use SilverStripe\GraphQL\MutationCreator;
+use SilverStripe\GraphQL\OperationResolver;
+use SilverStripe\ORM\DB;
+
+class SortBlockMutationCreator extends MutationCreator implements OperationResolver
+{
+    public function attributes()
+    {
+        return [
+            'name' => 'sortBlock',
+            'description' => 'Changes the sort position of an element'
+        ];
+    }
+
+    public function type()
+    {
+        return $this->manager->getType('Block');
+    }
+
+    public function args()
+    {
+        return [
+            'ID' => ['type' => Type::nonNull(Type::id())],
+            'AfterBlockID' => ['type' => Type::nonNull(Type::id())],
+        ];
+    }
+
+    public function resolve($object, array $args, $context, ResolveInfo $info)
+    {
+        $block = BaseElement::get_by_id($args['ID']);
+
+        if (!$block) {
```

Comment:
You also need to check for canEdit() with the `$context['currentUser']` ([docs](https://github.com/silverstripe/silverstripe-graphql#authentication))